source/includes/steps-view-logs.yaml

```diff
@@ -0,0 +1,83 @@
+---
+title: "Find your Session ID."
+level: 4
+ref: session-id
+content: |
+
+  .. example::
+
+     |mdb-shell| displays the session ID each time you open the shell.
+
+     .. code-block:: javascript
+        :copyable: false
+        :emphasize-lines: 3
+
+        $ mongosh
+
+        Current sessionID: c2961dbd6b73b052671d9df0
+        Connecting to: mongodb://127.0.0.1:27017
+        Using MongoDB: 4.2.8
+
+---
+title: "View the log for the session."
+level: 4
+ref: view-session-log
+content: |
+
+  |mdb-shell| saves the log for each session to your user's
+  ``.mongodb/mongosh`` directory:
+
+  .. tabs::
+
+     .. tab::
+        :tabid: macOS and Linux
+
+        .. code-block:: sh
+
+           ~/.mongodb/mongosh/<sessionId>_log
+
+        Run the following command to view the log for a session:
+
+        .. code-block:: sh
+
+           cat ~/.mongodb/mongosh/<sessionId>_log
+
+        Run the following command to tail the log for a session:
+
+        .. code-block:: sh
+
+           tail -f ~/.mongodb/mongosh/<sessionId>_log
+
+        .. tip::
+
+           You can pipe the output to an installed ``ndjson`` pretty
+           printer, such as `pino-colada
+           <https://www.npmjs.com/package/pino-colada>`__ 
+           or `pino-pretty <https://www.npmjs.com/package/pino-pretty>`__
+           to improve readability:
+
+           .. code-block:: sh
+
+              tail -f ~/.mongodb/mongosh/<sessionId>_log | pino-colada
+
+     .. tab::
+        :tabid: windows
+
+        .. code-block:: sh
+
+           %UserProfile%/.mongodb/mongosh/<sessionId>_log
+
+        Run the following PowerShell command to view the log for a session:
+
+        .. code-block:: sh
+
+           Get-Content %UserProfile%/.mongodb/mongosh/<sessionId>_log
+
+        Run the following PowerShell command to tail the log for a
+        session:
+
+        .. code-block:: sh
+
+           Get-Content %UserProfile%/.mongodb/mongosh/<sessionId>_log -Wait
```

Comment:
You can also pipe a pretty-printer on windows! For example with `pino-colada` you'd do the exact same pipe as on Linux or MacOS:
```shell
Get-Content %UserProfile%/.mongodb/mongosh/<sessionId>_log -Wait | pino-colada
```
<img width="991" alt="Annotation 2020-06-18 113317" src="https://user-images.githubusercontent.com/8107784/85004435-d5797780-b157-11ea-8f26-9c7548abafdf.png">

Reply:
updated